semantics/psc-trees-annotation.ads

```diff
@@ -54,6 +54,9 @@ package PSC.Trees.Annotation is
       New_Operand : Optional_Tree);
    --  Set Nth operand of given Tree
 
+   function Kind (T : Tree) return Tree_Kind_Enum;
```

Comment:
As mentioned below, this kind of simple function might best be an expression function:
`function Kind (T : Tree) return Tree_Kind_Enum is (Annotation_Kind);`